### DISCUSSION TITLE
Creating geopolars from polars

### DISCUSSION THREAD

**Discussion by ibbala** [pola-rs/geopolars#180] [Q&A] [answered]
Hi,
how to create a geopolars from a series/datafarme of polars? with geopandas we can do like this
```
df = pd.DataFrame(
    {'City': ['Buenos Aires', 'Brasilia', 'Santiago', 'Bogota', 'Caracas'],
     'Country': ['Argentina', 'Brazil', 'Chile', 'Colombia', 'Venezuela'],
     'Latitude': [-34.58, -15.78, -33.45, 4.60, 10.48],
     'Longitude': [-58.66, -47.91, -70.66, -74.08, -66.86]})
gdf = geopandas.GeoDataFrame(
    df, geometry=geopandas.points_from_xy(df.Longitude, df.Latitude))
```
Thank you in advance for your help

**Comment by kylebarron** [comment, marked as answer]
We don't currently have a way to construct a `GeoDataFrame` from a `DataFrame` plus geometries. Also most of the methods so far are implemented only for a `GeoSeries`. The easiest way is probably to construct a `geopandas.GeoSeries` and use `from_geopandas` to convert that to our `GeoSeries`.

**Comment by ibbala** [comment]
Thanks for ur reply and work

**Comment by arnovaill** [comment]
Hi there, any update on this ? I can convert my geodataframe (with a geoseries column with several 2D Points) to geopolars using  from_geopandas but after conversion this column shows: 'Binary data' instead of the coordinates. How could I convert a geoserie of 2D Points to geopolars ? Thanks a lot for your work!

**Reply by kylebarron** [reply]
> after conversion this column shows: 'Binary data' instead of the coordinates

That's because right now geopolars uses [Well-Known Binary](https://libgeos.org/specifications/wkb/) to store geometries, so they appear to be just a binary blob
